polly/lib/External/isl/imath/tools/findthreshold.py

```diff
@@ -86,9 +86,9 @@ def compute_stats():
     stats = compute_stats()
     stats.sort(key=lambda s: s[3] / s[2])
     for prec, thresh, trec, tnorm in stats:
-        print "%d\t%d\t%.3f\t%.3f\t%.4f" % (prec, thresh, trec, tnorm,
-                                            tnorm / trec)
+        print("%d\t%d\t%.3f\t%.3f\t%.4f" % (prec, thresh, trec, tnorm,
+                                            tnorm / trec))
```

Comment:
This change was already made upstream.
https://github.com/creachadair/imath/blob/f8197a2422a9c765d48331cc25aefe61b0cffb13/tools/findthreshold.py#L90-L91